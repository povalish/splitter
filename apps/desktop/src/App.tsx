import './tailwind.css';
import { Loader } from '@ui-kit/components/loader/Loader';
import { Button } from '@ui-kit/components/button/Button';

//
//

const App = () => (
  <div className='bg-slate-100 w-screen h-screen'>
    <h1 className='text-center text-xl text-black'>Hello from Electron!</h1>
    <Button>Click me!</Button>
    <Loader />
  </div>
);

export default App;
