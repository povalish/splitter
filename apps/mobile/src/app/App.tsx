import React from 'react';
import { SafeAreaView, StatusBar, Text, View } from 'react-native';
import { Loader } from '@ui-kit/components/loader/Loader.native';
import { Button } from '@ui-kit/components/button/Button.native';

//
//

export const App = () => (
  <>
    <StatusBar barStyle='dark-content' />
    <SafeAreaView>
      <Text testID='heading' className='text-2xl text-emerald-300'>
        Hello from ReactNative
      </Text>
      <View className='w-12 h-12 rounded-full bg-slate-300' />
      <Loader />
      <Button>Click me!</Button>
    </SafeAreaView>
  </>
);

export default App;
