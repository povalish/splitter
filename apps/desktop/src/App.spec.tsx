import { render } from '@testing-library/react';
import App from './App';

describe('App', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<App />);
    expect(baseElement).toMatchSnapshot();
  });

  it('should have a greeting as the title', () => {
    const { getByText } = render(<App />);
    expect(getByText('Hello from Electron!')).toBeTruthy();
  });
});
