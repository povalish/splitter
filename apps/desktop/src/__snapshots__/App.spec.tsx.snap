// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render successfully 1`] = `
<body>
  <div>
    <div
      style="background-color: red;"
    >
      <h1
        style="color: rgb(255, 255, 255);"
      >
        Hello from Electron!
      </h1>
      <button
        type="button"
      >
        Click me!
      </button>
    </div>
  </div>
</body>
`;
